### PR TITLE
Introduce tests for ck8s-cli

### PR DESCRIPTION
**Pull request by aczajkowski**
First implementation of test for CLI

### CODE REVIEW

**Review by brig — APPROVED** [review]

File: cli/src/main/java/brig/ck8s/cli/CliApp.java [hunk 1]
```diff
@@ -68,7 +68,7 @@ public class CliApp
             "-VVV debug logs"})
     boolean[] verbosity = new boolean[0];
 
-    @CommandLine.Option(names = {"-t"}, description = "Test mode: Only display the command that will be executed")
+    @CommandLine.Option(names = {"-t", "--dry-run"}, description = "Test mode: Only display the command that will be executed")
```
Comment (ppalucha) [inline cli/src/main/java/brig/ck8s/cli/CliApp.java:71]: nit: separate commit

File: cli/src/main/java/brig/ck8s/cli/Main.java [hunk 2]
```diff
@@ -26,10 +26,10 @@ public class Main
 {
 
     public static void main(String[] args)
-            throws Exception
     {
         CommandLine cmd = new CommandLine(new CliApp())
-                .setCaseInsensitiveEnumValuesAllowed(true);
+                .setCaseInsensitiveEnumValuesAllowed(true)
```
Comment (ppalucha) [inline cli/src/main/java/brig/ck8s/cli/Main.java:31]: nit: separate commit

File: cli/src/test/java/brig/ck8s/cli/assertions/CliExec.java [hunk 3]
```diff
@@ -0,0 +1,107 @@
+/*
```
Comment (ppalucha) [inline cli/src/test/java/brig/ck8s/cli/assertions/CliExec.java]: !

**Review by ppalucha — APPROVED** [review]

File: cli/src/main/java/brig/ck8s/cli/CliApp.java (hunk 1 above)
Comment (aczajkowski) [reply cli/src/main/java/brig/ck8s/cli/CliApp.java:71]: Applied

File: cli/src/main/java/brig/ck8s/cli/Main.java (hunk 2 above)
Comment (aczajkowski) [reply cli/src/main/java/brig/ck8s/cli/Main.java:31]: Applied

**Pull request merged**